lib/streamlit/hello/pages/2_Mapping_Demo.py

```diff
@@ -55,7 +55,7 @@ def from_data_file(filename):
                 get_position=["lon", "lat"],
                 get_text="name",
                 get_color=[0, 0, 0, 200],
-                get_size=15,
+                get_size=10,
                 get_alignment_baseline="'bottom'",
```

Comment:
Clarification note: The text label size was much larger in the mapping demo as indirect result of dependency updates - this fixes the inadvertent change in text size to mirror existing.